### ISSUE TITLE
Uncaught TypeError: Cannot read property 'onDidDestroy' of undefined

### ISSUE THREAD

**Issue by michael-yx-wu** [steelbrain/linter#1015]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.3.0
**System**: Mac OS X 10.11.1
**Thrown From**: [linter](https://github.com/atom-community/linter) package, v1.11.3

### Stack Trace

Uncaught TypeError: Cannot read property 'onDidDestroy' of undefined

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/decoration.js:58

TypeError: Cannot read property 'onDidDestroy' of undefined
    at new Decoration (/Applications/Atom.app/Contents/Resources/app.asar/src/decoration.js:58:49)
    at DisplayBuffer.module.exports.DisplayBuffer.decorateMarker (/Applications/Atom.app/Contents/Resources/app.asar/src/display-buffer.js:1052:20)
    at TextEditor.module.exports.TextEditor.decorateMarker (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:1274:33)
    at LinterViews.renderBubble [as _renderBubble] (/Users/michaelwu/.atom/packages/linter/lib/linter-views.js:86:43)
    at /Users/michaelwu/.atom/packages/linter/lib/helpers.coffee:23:18
```

### Commands

```
  4x -3:55.3.0 window:focus-pane-on-right (atom-text-editor.editor.is-focused)
     -3:49.3.0 settings-view:open (atom-text-editor.editor.is-focused)
     -3:48.2.0 core:close (div.panels-item)
     -3:29.7.0 pane:show-previous-item (atom-text-editor.editor.is-focused)
     -3:27.7.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
     -3:26.1.0 window:focus-pane-on-right (atom-text-editor.editor.is-focused)
     -2:54.5.0 pane:split-right (atom-text-editor.editor.is-focused)
     -0:37.4.0 core:move-down (atom-text-editor.editor.autocomplete-active.is-focused)
     -0:37.2.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.autocomplete-active.is-focused)
     -0:35.5.0 editor:delete-to-beginning-of-word (atom-text-editor.editor.autocomplete-active.is-focused)
  2x -0:34.8.0 pane:show-previous-item (atom-text-editor.editor.autocomplete-active.is-focused)
  2x -0:34 pane:show-next-item (atom-text-editor.editor.is-focused)
  2x -0:33.6.0 pane:show-previous-item (atom-text-editor.editor.autocomplete-active.is-focused)
  2x -0:33.1.0 core:close (atom-text-editor.editor.is-focused)
     -0:31.3.0 window:focus-pane-on-right (atom-text-editor.editor.autocomplete-active.is-focused)
     -0:04.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "packagesWithKeymapsDisabled": [
      "rst-preview"
    ],
    "disabledPackages": [
      "linter-pep8"
    ]
  },
  "linter": {}
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
atom-typescript, v7.14.2
autocomplete-python, v1.2.3
language-docker, v1.1.6
language-gradle, v0.0.3
language-groovy, v0.4.0
language-restructuredtext, v0.11.0
linter, v1.11.3
linter-clang, v3.3.0
linter-javac, v1.3.0
linter-pylint, v1.2.0
linter-shellcheck, v1.0.2
minimap, v4.17.0
minimap-find-and-replace, v4.4.0
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-selection, v4.3.1
omni-ruler, v0.4.2
rest-client, v0.5.0
rst-preview-pandoc, v0.1.12
sync-settings, v0.6.0

# Dev
No dev packages
```

**Comment by steelbrain** [comment]
@michael-yx-wu that bug you've hit is mysterious, can you please provide a repro case?

**Comment by uxl** [comment]
Using linter with jslint .. if you split panel to the right and the script in that second panel has lint errors I get a series of these bugs notifications.

**Comment by michael-yx-wu** [comment]
I was using the linter-pylint package and had specified an invalid path to my .pylintrc file in my project. After I fixed the path, these errors started going away. Still not really sure why this occurred, but thought I should flag it.

**Comment by steelbrain** [comment]
> linter-pylint

cc @SpainTrain

**Comment by WillsB3** [comment]
Also have this using linter and linter-gjslint, I too had split pane and the right file causes these errors.

**Comment by Arcanemagus** [comment]
This should be fixed by the UI re-write that is being worked on.

**Comment by steelbrain** [comment]
Fixed in v1.11.5
